official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -76,22 +76,17 @@ def benchmark_8_gpu(self):
     FLAGS.enable_eager = True
     self._run_and_report_benchmark()
 
-  def benchmark_8_gpu_bfc_allocator(self):
-    """Restricts CPU memory allocation."""
+  def benchmark_8_gpu_tweaked(self):
```

Comment:
Can you  go ahead and delete this test completely and make the test above `def benchmark_8_gpu(self):`  run with the 14 threads.  Since the goal for this class of tests is to test accuracy first it is ok for the accuracy test to be "tuned".  The short test are our primary location for perf tests.  I realize this is a bit weird.  If the accuracy tests did not take 16 hours I would have a normal and then tweaked version.  Thank you and this is a great test to track our progress.

Reply:
Done.